galaxykit/client.py

```diff
@@ -74,8 +74,8 @@ def __init__(
                     or urlparse(self.galaxy_root).netloc.split(":")[0] + ":5001"
                 )
 
-                self.docker_client = dockerutils.DockerClient(
-                    (username, password),
+                self.container_client = containerutils.ContainerClient(
+                    (self.username, self.password),
```

Comment:
It doesn't look like these attributes exist, they're only passed in as parameters.

Reply:
ahhh good catch! Those are some stray changes for another feature that got into this PR.

Reply:
Fix pushed.